docs/getting_oss_rules_of_play_rating.md

```diff
@@ -18,6 +18,11 @@ The data provider `ContributingGuidelineInfo` allows you to specify 2 types of r
 - The content that you expect must not be part of the contributing guidelines.
 The option is called `requiredContentPatterns` and expects a list of one or multiple Regex. If you enter multiple entries, the contribution guidelines must comply with all the patterns for the rule to pass.
 
+### Code of Conduct Guidelines
+The data provider `CodeOfConductGuidelineInfo` allows you to specify 1 type of regular expression pattern
+- The content that you expect to be part of the code of conduct guidelines.
+The option is called `requiredContentPatterns` and expects a list of one or multiple Regex. If you enter multiple entries, the code of conduct guidelines must comply with all the patterns for the rule to pass.
```

Comment:
I think if this is put as parameter un config.yml.

Then the system would not know if it is from Code of Conduct Guidelines or Contributing Guideline Info.

I think we need to change the attribute name in the program and also in the config.yml, test files

Reply:
When we create a config object we use the `.class.getSimpleName()`  which return the classname and we append the `.config.yml` to it at the end. We also have 2 different files 

 - CodeOfConductGuidelineInfo.config.yml
 - ContributingGuidelineInfo.config.yml

Which both contain attribute `requiredContentPatterns` so even the attribute name is same the files are different so there shouldn’t be an confusion.

Reply:
I checked this. Thanks for the clarification.